univocity-trader-core/src/main/java/com/univocity/trader/indicators/RAVIIndicator.java

```diff
@@ -0,0 +1,44 @@
+package com.univocity.trader.indicators;
+
+import com.univocity.trader.candles.Candle;
+import com.univocity.trader.indicators.base.SingleValueIndicator;
+import com.univocity.trader.indicators.base.TimeInterval;
+import com.univocity.trader.strategy.Indicator;
+
+public class RAVIIndicator extends SingleValueIndicator {
+
+    private MovingAverage shortSma;
+    private MovingAverage longSma;
+
+    private double value;
+
+    public RAVIIndicator(int shortSmaBarCount, int longSmaBarCount, TimeInterval timeInterval) {
+        super(timeInterval, null);
```

Comment:
If sensible, provide the value getter function to the `shortSma` and `longSma` indicators instead of null. Allow for another constructor that uses a function that returns a the default `candle.close` and pass that function over to the indicators. Take a look at the `BollingerBand` implementation with its multiple constructors.